Messaging.md

```diff
@@ -262,10 +291,9 @@ all of the `negotiation_fields` nested here must be version 0 or 1.
 
 This type contains information about a specific input to be used in a funding transaction, as well as its corresponding on-chain UTXO.
 
-#### Version 0 `funding_input`
+#### `funding_input`
 
-1. type: 42772 (`funding_input_v0`)
-2. data:
+1. data:
    * [`u64`:`input_serial_id`]
    * [`u16`:`prevtx_len`]
```

Comment:
I think this is a `bigsize` now, at least according to the `enum_single_oracle_test.json`

Reply:
Yes thanks fixed.